packages/app/src/app/pages/Sandbox/Editor/Header/SandboxName/SandboxName.tsx

```diff
@@ -117,7 +118,14 @@ export const SandboxName = observer(() => {
             />
           </Form>
         ) : (
-          <Name onClick={handleNameClick}>{sandboxName}</Name>
+          <Name
+            css={`
```

Comment:
Name is imported from elements.js next to this file, right? how aobut we pass a prop to change the cursor state instead of using inlined css here?

Reply:
done